system-x/services/azure/azure-common/src/main/java/software/tnb/azure/common/account/AzureAccount.java

```diff
@@ -1,9 +1,9 @@
-package software.tnb.azure.storage.blob.account;
+package software.tnb.azure.event.hubs.account;
```

Comment:
incorrect package name, should be `software.tnb.azure.common.account`

Reply:
ups, changed